src/esp/gfx/RenderingTarget.cpp

```diff
@@ -0,0 +1,190 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+#include <Magnum/GL/Buffer.h>
+#include <Magnum/GL/DefaultFramebuffer.h>
+#include <Magnum/GL/Framebuffer.h>
+#include <Magnum/GL/Renderbuffer.h>
+#include <Magnum/GL/RenderbufferFormat.h>
+#include <Magnum/GL/Renderer.h>
+#include <Magnum/GL/Texture.h>
+#include <Magnum/GL/TextureFormat.h>
+#include <Magnum/Image.h>
+#include <Magnum/PixelFormat.h>
+
+#include "RenderingTarget.h"
+#include "magnum.h"
+
+using namespace Magnum;
+
+namespace esp {
+namespace gfx {
+struct RenderingTarget::Impl {
+  Impl(WindowlessContext::ptr context, int height, int width)
```

Comment:
I'm sure there's some convention I'm not aware about but ... why is it height first? :) What about using a `Vector2i` for that instead to make the argument order clear? (Instantiating the class would be then just a matter of wrapping the args in `{width, height}`, or, in case of python, `[width, height]`, nothing verbose).

Reply:
height, width is common when treating images an array (which is super common for computer vision people) -- I originally had it as a vector, but kept in passing `[height, width]` and breaking things....

I can change to width, height in the constructor, but I will keep them separate so I don't have to debug that again :)

Reply:
OK, was not aware of that. Then keep it in whatever form is the least surprising for the target audience :)

Reply:
I agree with @mosra . W x H is the convention for computer graphics (e.g., 1920 x 1080). Also please note that we have "width" ahead of "height" almost everywhere in the C++ backend. I am not against using H x W, but please be careful when you work on the code bridging the python and C++.

Reply:
I used H x W here as we pass the sensor resolutions as H x W https://github.com/facebookresearch/habitat-sim/blob/master/examples/settings.py#L43

Reply:
> I used H x W here as we pass the sensor resolutions as H x W https://github.com/facebookresearch/habitat-sim/blob/master/examples/settings.py#L43

I will advise in pybindings which are exposed to the external users (main audience perhaps are from CV community), you use H X W, while in the C++ backend infra (the simulator), you use W X H. 

Does it make sense?